server/portal/apps/projects/management/commands/migrate-projects.py

```diff
@@ -17,35 +18,36 @@ class Command(BaseCommand):
     help = (
         'Reconcile projects from CEPv1 migration that do not have '
         'a PI assigned. This will attempt to assign the creator of '
-        'these projects as PI.'
+        'these projects as PI, as well as index all projects.'
     )
 
     def handle(self, *args, **options):
         """Handle command."""
         agc = service_account()
-        metadatas = ProjectMetadata.objects.all().filter(pi=None)
+        metadatas = ProjectMetadata.objects.all()
         for meta in metadatas:
-            try:
-
-                project = Project(agc, meta.project_id)
-                storage = project.storage
-                roles = storage.roles.to_dict().items()
-                admins = list(
-                    filter(
-                        lambda role_tuple: role_tuple[0] != 'wma_prtl' and (role_tuple[1] == 'ADMIN' or role_tuple[1] == 'OWNER'),
-                        roles
+            index_project.apply_async(args=[meta.project_id])
```

Comment:
Might we also want to move the index task `index_project.apply_async(args=[meta.project_id])` to take place after the `if not meta.pi:` block to make sure the metadata index takes into account any changes made in the block?